canonical/charmhub.io
# Release beta store

## Done
Made the beta store the default homepage

## QA
- Go to https://charmhub-io-1674.demos.haus/
- Click around and make sure it works as expected
